commands/commands.go

```diff
@@ -334,6 +335,46 @@ func logPanic(loggedError error) string {
 	return full
 }
 
+func ipAddresses() []string {
+	ips := make([]string, 0, 1)
+	ifaces, err := net.Interfaces()
+	if err != nil {
+		ips = append(ips, "Error getting network interface: "+err.Error())
+		return ips
+	}
+	for _, i := range ifaces {
+		if i.Flags&net.FlagUp == 0 {
+			continue // interface down
+		}
+		if i.Flags&net.FlagLoopback != 0 {
+			continue // loopback interface
+		}
+		addrs, _ := i.Addrs()
+		s := ""
+		if err != nil {
+			ips = append(ips, "Error getting IP address: "+err.Error())
+			continue
+		}
+		for _, addr := range addrs {
+			var ip net.IP
+			switch v := addr.(type) {
+			case *net.IPNet:
+				ip = v.IP
+			case *net.IPAddr:
+				ip = v.IP
+			}
+			if ip == nil || ip.IsLoopback() {
+				continue
+			}
+			s = s + ip.String() + " "
```

Comment:
Instead of adding a `" "` on the end of `s` here, could you use a `[]string` and call `strings.Join(slice, " ")` to prevent the trailing whitespace?

Reply:
Good idea!